ffi/src/sspi/sec_pkg_info.rs

```diff
@@ -22,8 +22,10 @@ pub struct SecPkgInfoW {
 
 pub type PSecPkgInfoW = *mut SecPkgInfoW;
 
+pub struct RawSecPkgInfoW(pub *mut SecPkgInfoW);
```

Comment:
question/nitpick: Should we use the `PSecPkgInfoW` type alias? Remove it?

Reply:
We are not allowed to implement the `From` trait for any raw pointers, because all raw pointers are foreign types

Reply:
Sorry, I meant this:

```suggestion
pub struct RawSecPkgInfoW(pub PSecPkgInfoW);
```

I am under the impression that we don’t really use this alias.